### ISSUE TITLE
Downloaded pose.pdb erroneously have all ligands

### ISSUE THREAD

**Issue by phraenquex** [m2ms/fragalysis-frontend#1480]

**Comment by mwinokan** [comment]
I just took a look at a fresh download of FatA from staging and the pdb's for x1201a and x1201b do indeed both have two ligands (although the protein conformations are not identical). 

I will talk to Kate, because I don't think XCA is being applied correctly in the first place, in the download and Fragalysis there are still two distinct protein units/chains, even though I'm assuming they are supposed to be aligned onto one.

**Comment by phraenquex** [comment]
@mwinokan: Kate also showed me her question.  I conclude it probably _is_ a bug in the XCA spec (likely my own) 

New spec:  for each pose, all non-site _ligands_ should be stripped out. 

However, keep the co-factors and waters and metals - everything not named LIG or LG1, LG2 ,..

(If controversial, I can explain more;  and tell me if I've missed critical corner cases.)

**Comment by mwinokan** [comment]
Yes that seems like a reasonable change to the XCA spec. Should it be green?

**Comment by phraenquex** [comment]
Yes, because it's a data-breaking change.  

(We can reconsider if @tdudgeon or @ConorFWild say it's Very Big Ticket.)

**Comment by tdudgeon** [comment]
This could be a bit tricky if I understand it correctly. Until we get the combi-soak work done (#1432) then it's probably going to be difficult to know what the "real" ligand is and what should be removed. Also, it's not clear to me whether this just impacts the post-processing of the aligned structures or whether the alignment algorithm need to change to exclude the undesired ligands.

@mwinokan do you have a concrete example we can look at (preferably in the 3 datasets that we have been working with) and a description of what the desired behaviour should be?

**Comment by mwinokan** [comment]
Thanks @tdudgeon, it makes sense that this ticket would be immensely aided by completed the combi-soak work first.

As far as test data I'm not currently aware of any such issues in the 3 datasets from #1456. I suggest we hold our horses for the moment and see if the FatA issues survive a fresh alignment by Kate (she had last aligned before the #1457 fix)

**Comment by mwinokan** [comment]
@tdudgeon and @ConorFWild suspect that this is an alignment issue. @phraenquex will ask Kate to share a snapshot with the issue

@phraenquex says to prune ligands on the same chain but in different CanonicalSites from the one that is being aligned

**Comment by phraenquex** [comment]
Everything that's NOT aligned to the canonical site should be removed.  @ConorFWild confirms it's a LNA problem. 

@ConorFWild and @phraenquex to finalise.

**Comment by mwinokan** [comment]
Providing the correct example data that is problematic, @ConorFWild says he is able to fix this and #1477 quite quickly.

**Comment by mwinokan** [comment]
@Waztom I've uploaded [2A to Kalev's stack ](https://fragalysis-kalev-default.xchem-dev.diamond.ac.uk/viewer/react/preview/target/A71EV2A/tas/lb32627-66)

**Comment by ConorFWild** [comment]
Waiting on example data then will fix: please link it here

**Comment by mwinokan** [comment]
@ConorFWild I have not heard back from Kate but I believe x1201 exhibits this issue. There is one ligand in each chain, and the Fragalysis download contains both ligands in each x1201a and x1201b observation PDBs.

`/dls/labxchem/data/lb36049/lb36049-1/processing/lb30602-66/processing/analysis/model_building/FATAATA-x1201`

**Comment by mwinokan** [comment]
@ConorFWild Kate has confirmed the above directory

**Comment by mwinokan** [comment]
Oops, the path in my last comment had an extra `analysis`. 

@ConorFWild does the following directory have what you need?
```
/dls/labxchem/data/lb36049/lb36049-1/processing/lb30602-66/processing/analysis/outputs
```

**Comment by mwinokan** [comment]
@ConorFWild says that a fix has been implemented but needs checking and the cluster is currently down.

**Comment by mwinokan** [comment]
@ConorFWild believes this to be implemented, and will be merged together with #1477.

@ConorFWild still needs the data and config.yaml paths from Kate. @mwinokan will put them in touch via slack

**Comment by mwinokan** [comment]
@ConorFWild says it's likely this will be fixed today. There was a typing bug that has been fixed.

**Comment by ConorFWild** [comment]
Changes merged!

**Comment by phraenquex** [comment]
Needs testing - @mwinokan ask Kate Kot.  CHIKV might also flush this out.
